src/main/java/com/spotify/sparkey/Sparkey.java

```diff
@@ -75,6 +76,17 @@ public static SparkeyWriter appendOrCreate(File file, CompressionType compressio
     return SingleThreadedSparkeyWriter.appendOrCreate(file, compressionType, compressionBlockSize);
   }
 
+  /**
+   * Open a new, thread-safe, sparkey reader
+   *
+   *
+   * @param file File base to use, the actual file endings will be set to .spi and .spl
+   * @return a new reader,
+   */
+  public static SparkeyReader open(File file) throws IOException {
```

Comment:
Also add a `openThreadLocalReader` method in this class

Reply:
okay. BTW, what's the benefit of exposing a non thread safe reader at all? Does the synchronization make ThreadLocal somewhat more expensive?

Reply:
Yes, for some use cases you may want to control the thread safety manually and avoid the extra overhead